test/test_operators.py

```diff
@@ -18,6 +18,7 @@
 from cosmic_ray.operators.break_continue import (ReplaceBreakWithContinue,
                                                  ReplaceContinueWithBreak)
 from cosmic_ray.operators.number_replacer import NumberReplacer
+from cosmic_ray.operators.exception_replacer import ExceptionReplacer
```

Comment:
Put this in alphabetical order with the other imports. (Sorry, I'm a bit picky about some things).